### ISSUE TITLE
Wrong version of php-cs-fixer installed

### ISSUE THREAD

**Issue by dktapps** [shivammathur/setup-php#485]
**Describe the bug**
It looks like the action always uses the latest release of php-cs-fixer, which isn't always the newest one, since they maintain 2.19 and 3.0 simultaneously.

This leads to php-cs-fixer 3.0-dependent runs breaking spontaneously in the last day or two because the action suddenly started providing 2.19 instead of 3.0, since 2.19.2 is the newest release and not 3.0.2.

**Version**
2.12.0

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**
ubuntu-20.04

**PHP versions**
8.0

**To Reproduce**

Seems this is all that's required: https://github.com/pmmp/PocketMine-MP/blob/stable/.github/workflows/main.yml#L279-L283

**Expected behavior**
php-cs-fixer 3.0.x should be downloaded since that's what it's been providing for weeks.

**Screenshots/Logs**
https://github.com/pmmp/PocketMine-MP/runs/3367235033?check_suite_focus=true#step:3:12

**Additional context**

**Are you willing to submit a PR?**

**Comment by shivammathur** [comment]
Please ask upstream to set 3.0.2 as the latest version in their releases.

Also, you can specify the major version to 3 and specify a GitHub OAuth token with COMPOSER_TOKEN env.
https://github.com/shivammathur/setup-php#wrench-tools-support

```yml
- name: Setup PHP and tools
  uses: shivammathur/setup-php@2.12.0
  env:
    COMPOSER_TOKEN: ${{ GITHUB_TOKEN }}
  with:
    php-version: 8.0
    tools: php-cs-fixer:3
```

If this is not fixed upstream, I will try to find a resolution for this in the next release, but most likely will have to switch to installing php-cs-fixer using composer instead of release phars, which is slower.

**Comment by dktapps** [comment]
I can file the issue, but it's likely a similar problem will crop up again since they regularly release multiple versions simultaneously. It's hard to avoid.

**Comment by shivammathur** [comment]
Fixed in https://github.com/shivammathur/setup-php/commit/4d3cfaa6531effaa5068d7e54df6037e96c09aa8, will be in next release.

**Comment by dktapps** [comment]
Awesome, thank you.

**Comment by shivammathur** [comment]
Fixed in 2.13.0
Test workflow: https://github.com/shivammathur/test-setup-php/actions/runs/1158374085/workflow
